src/sat/boolean_problem.cc

```diff
@@ -232,12 +233,13 @@ void UseObjectiveForSatAssignmentPreference(const LinearBooleanProblem& problem,
   const LinearObjective& objective = problem.objective();
   double max_weight = 0;
   for (int i = 0; i < objective.literals_size(); ++i) {
-    max_weight = std::max(max_weight,
-                          fabs(static_cast<double>(objective.coefficients(i))));
```

Comment:
std::abs is already used. This red snippet is the removed code, see the green code snippet below for what is appended. It already uses std::fabs. (actually, this is what this commit was about. :-))